README.md

```diff
@@ -63,16 +44,74 @@ From there, it does a couple of things:
     - rationale for this is to give us granular control over certain units, such as mesos-slaves. It allows us to control individual nodes, or perform rolling actions (such as deploys) while retaining visibility into the cluster as a whole.
 6. submits and starts generic fleet units
 
-#### Docker Image ID etcd keys (under `/images`):
+Services
+====
+
+### Global Services (run on ALL nodes in ALL tiers)
+
+#### Monitoring
+  - [Datadog](https://www.datadoghq.com/)
+  - [Sysdig](http://www.sysdig.org/)
+  - [Sumologic](https://www.sumologic.com/)
+
+#### Util/Automated Maintenance
+  - Docker Logrotate (based on [michaloo/logrotate](https://github.com/michaloo/logrotate))
+  - Docker Image/Container Cleanup
+
+#### MISC
+  - SSHD mask
+      - [bug in CoreOS](https://github.com/coreos/bugs/issues/966)
+      - [proposed changes](https://github.com/coreos/init/pull/188)
+
+### Control Tier Nodes:
+
+- [Mesos](http://mesos.apache.org/) Master
+- [Marathon](https://mesosphere.github.io/marathon/)
+- [Exhibitor (for Zookeeper)](https://github.com/Netflix/exhibitor)
+- [Chronos](http://mesos.github.io/chronos/)
+- [Flight Director](https://github.com/adobe-platform/flight-director) - private Marathon deployment wrapper/manager (stay tuned!)
+- [HUD](https://github.com/adobe-platform/flight-director-hud) - private UI shim for flight-director (stay tuned!)
+
+### Proxy Tier Nodes:
+- [`CAPCOM`](https://github.com/adobe-platform/capcom) - private Container-Proxy Manager (stay tuned!)
+- Heatshield Proxy (our version of nginx) or HAProxy
+
+### Worker Tier Nodes:
+- Mesos Slave
+
+Key/Secret Management & Configuration
+====
+
+All secrets & key management is a bit adhoc. Most of the `setup` scripts, which house the logic for setting up the data for then fleet units to use, require a few things to download secrets & keys:
+
+- the `$CONTROL_TIER_S3SECURE_BUCKET` environment variable, written into `/etc/environment` by cloudformation
+- [behance/docker-aws-s3-downloader](https://github.com/behance/docker-aws-s3-downloader) container to download files
+- IAM roles to access `$CONTROL_TIER_S3SECURE_BUCKET`
+
+Secrets make it onto the nodes in the form of flat text files that live within `$CONTROL_TIER_S3SECURE_BUCKET`. The `setup` files **individually** know which file(s) they need to download & how to read, set or use the data for their corresponding units. So for example, the [datadog unit](https://github.com/adobe-platform/mesos-systemd/blob/master/v2/fleet/datadog.service#L21) requires an `etcd` key, `/ddapikey`. Knowing this, we have a [datadog setup script](https://github.com/adobe-platform/mesos-systemd/blob/master/v2/setup/datadog.sh) which downloads a `.datadog` file from `$CONTROL_TIER_S3SECURE_BUCKET`, expects it to be in a certain format, and sets the etcd key.
+
+#### Files in S3 (deprecated)
```

Comment:
Can you add what we plan to replace it with? (exploring KMS-encrypted dynamo table)